packages/flutter/lib/src/material/mergeable_material.dart

```diff
@@ -471,10 +479,37 @@ class _MergeableMaterialState extends State<MergeableMaterial> {
     );
   }
 
+  List<Widget> _divideSlices(BuildContext context, List<Widget> slices, List<Key> keys) {
+    if (config.hasDividers) {
+      List<Widget> divided = <Widget>[];
+
+      for (int i = 0; i < slices.length; i += 1) {
+        divided.add(
+          new DecoratedBox(
+            key: keys[i],
+            decoration: i != slices.length - 1 ? new BoxDecoration(
+              border: new Border(
+                bottom: new BorderSide(color: Theme.of(context).dividerColor)
```

Comment:
It should be `1.0`, AFAIK. This is the same approach as in [`divideItems`](https://docs.flutter.io/flutter/material/ListItem/divideItems.html).